### PR TITLE
CRM457-1364 & CRM457-1454

### PR DESCRIPTION
**Pull request by ivanELEC**
## Description of change

 [CRM4 Provider: update file size hint text](https://dsdmoj.atlassian.net/browse/CRM457-1364)

[CRM7 Provider: amendment to upload supporting evidence page warning text](https://dsdmoj.atlassian.net/browse/CRM457-1454)

Note: CRM457-1454 doesn't specifically reference it but the "Warning" text shows because we were using the wrong class for screen reader only text, I updated the class instead of removing the text in all occurences

### CODE REVIEW

File: config/locales/en/prior_authority/steps.yml [hunk 1]
```diff
@@ -173,9 +174,9 @@ en:
           page_title: Why is prior authority required?
           reason_why_hint_p1: 'Provide a summary of:'
           reason_why_hints:
-            - why you are requesting this authority
-            - details of the prosecution case
-            - the defence or mitigation
+            - Why you are requesting this authority
```
[the remark this answers is not in the document]
Comment (ivanELEC) [reply config/locales/en/prior_authority/steps.yml:177]: yes you're right, I misread the ticket. Will revert back

**Review by imtiazAhmd — APPROVED** [review]
LGTM

**Review by patrick-laa — APPROVED** [review]
Given we want to show an identical message in each case I wonder if it might be a marginal improvement to have it defined once in config/locales/en/prior_authority.yml under "generic.file_upload_hint" or something. But not too fussed either way.

**Comment by ivanELEC** [comment]
> Given we want to show an identical message in each case I wonder if it might be a marginal improvement to have it defined once in config/locales/en/prior_authority.yml under "generic.file_upload_hint" or something. But not too fussed either way.

If this was on the caseworker app I'd agree but I feel like because on the provider side the audiences are different I want to try and keep this kind of thing still per form type/service so that if different changes are needed on different services depending on feedback we don't have to "un-unify" things

**Pull request merged**